data/_demodata/applications/datacatalogue/Collection events.csv

```diff
@@ -1,62 +1,62 @@
-testCohort1,test collection event 1A,test description for test collection event 1A,test subcohort 1A,1958,4,1994,4,"All ages,Infant (0-23 months),Newborn (0-1 months),Infants and toddlers (2-23 months),Child (2-12 years),Adolescent (13-17 years),Adult (18+ years),Young adult (18-24 years),Adult (25-44 years),Middle-aged (45-64 years),Aged (65+ years),Aged (65-79 years),Aged (80+ years)",3500,"Laboratory measures,Hematology,Biochemistry,Microbiology,Virology,Immunology,Toxicology,Histology,Genomics,Other laboratory measures","Medical records,National registries,Genealogical records","DNA,Plasma,RNA",Abstinence Orientation Scale,test other standardised tools for collection event 1A,"test core variable 1 for collection event 1A,test core variable 2 for collection event 1A,test core variable 3 for collection event 1A",test supplementary information for collection event 1A blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah  UNTIL NOW,1958-04-01,30,1994-04-30
+resource,name,description,subpopulations,start date,end date,age groups,number of participants,areas of information,data categories,sample categories,standardized tools,standardized tools other,core variables,contact point.resource,contact point.first name,contact point.last name,publisher.resource,publisher.id,creator.resource,creator.id,license,issued,modified
+EDEN,20-24 week's of amenorrhoea (WA),This is a test description,mothers,,,,,,,,,,,EDEN,Bob,Ross,EDEN,INSERM,EDEN,INSERM,https://placeholder-license.nl,1990-01-01 00:00:00.0,1990-01-01 00:00:00.0
```

Comment:
we hebben vaak een 'start date' dus dan kan de license daar toch niet voor liggen?

Reply:
Is het niet logischer om even te kijken wanneer wij zijn begonnen met deze versie van de catalogus en die datum te kiezen?

Reply:
Oh sorry, ik ben veel te serieus nu, dit is demo data

Reply:
maar voor de 'echte' data geld dit denk ik wel